prodigy/datasets/datasetCreation.py

```diff
@@ -0,0 +1,131 @@
+import json
+import psycopg2
```

Comment:
Please follow the python style guide (e.g. dataset_creation instead of datasetCreation).